src/citadel/citadel.utils.ts

```diff
@@ -28,6 +28,7 @@ import {
 } from './citadel.constants';
 import { CitadelKnightsRoundStat } from './interfaces/citadel-knights-round-stat.interface';
 import { getFullToken } from '../tokens/tokens.utils';
+import * as citadelUtils from './citadel.utils';
```

Comment:
This is just for tests mock boundaries?

Reply:
yes, u can mock only imports, so if one function utilize another from same module, jest unable to mock that links